src/elm/clinical.ts

```diff
@@ -103,11 +111,12 @@ export class CodeSystemDef extends Expression {
   id: string;
   version: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.CodeSystemDef) {
+    // @ts-ignore
```

Comment:
Same as initial comment with more explanation of ts-ignore and disabling the lint rule